pyproject.toml

```diff
@@ -178,13 +179,15 @@ test = "pytest test --cov=tianshou --cov-report=xml --cov-report=term-missing --
 test-reduced = "pytest test/base test/continuous --cov=tianshou --durations=0 -v --color=yes"
 _black_check = "black --check ."
 _ruff_check = "ruff check ."
+_ruff_check_nb = "nbqa ruff docs/02_notebooks"
```

Comment:
This is problematic b/c it's used in `poe lint` which is not supposed to format anything. However, this command actually formats the notebooks (even without the `--fix`). I know that `nbqua "ruff --check ."` didn't work for a mysterious reason, but I wouldn't want to modify the logic of lint. 

If you don't find a fix for  `nbqua "ruff --check .", we could introduce a new poe command `nb-lint-format` and call in in CI additionally to `poe lint`. It's not nice, but better than changing `lint` semantics. Then pls open an issue in the nbqa repository describing the problem and add a block comment here that links to the issue